evalml/tests/conftest.py

```diff
@@ -48,6 +48,7 @@ def pytest_configure(config):
         "markers",
         "skip_offline: mark test to be skipped if offline (https://api.featurelabs.com cannot be reached)",
     )
+    config.addinivalue_line("markers", "noncore_dependency: mark test as slow to run")
```

Comment:
Nice! Is the reason why we mark these tests because they're slow to run? Do we want to mention instead that they contain dependencies that are not part of the core requirements instead?

Reply:
This is what happens when you copy-paste from the pytest docs 😂 Will update the message.